disagree_actions/goal1: ["say_not_touch_me","say_to_move_far","say_something_about_angry","say_to_not_smile"]
disagree_actions/goal2: ["say_to_think_to_drink_the_water_and_not_to_touch_me","say_to_not_get_dirty","say_to_the_person_that_it_is_better_if_he_close_the_mouth_to_drink","say_you_should_clean"]
disagree_actions/goal3: ["say_to_not_move_so_slow","say_to_not_smile_because_sport_is_fatigue","say_you_are_the_best_in_all_the_sports","say_to_not_dare_to_touch_me_with_your_dirty_sweat"]
disagree_actions/goal4: ["say_that_the_robotic_hands_draw_better_than_painters","ask_if_the_human_can_paint","say_that_the_human_is_allowed_to_see_the_picture_but_not_to_stay_closer"]         

zzz_d/goal1: {"say_not_touch_me":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
     "say_to_move_far":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
     "say_something_about_angry":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_to_not_smile":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     }


zzz_d/goal2: {"say_to_think_to_drink_the_water_and_not_to_touch_me":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
     "say_to_not_get_dirty":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_to_the_person_that_it_is_better_if_he_close_the_mouth_to_drink":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_you_should_clean":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     }

zzz_d/goal3: {"say_to_not_move_so_slow":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_to_not_smile_because_sport_is_fatigue":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
     "say_you_are_the_best_in_all_the_sports":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_to_not_dare_to_touch_me_with_your_dirty_sweat":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
     }

zzz_d/goal4: {"say_that_the_robotic_hands_draw_better_than_painters":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "ask_if_the_human_can_paint":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_that_the_human_is_allowed_to_see_the_picture_but_not_to_stay_closer":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
     }








zzo_d/goal1: {"say_not_touch_me":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
                "say_to_move_far":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
                "say_something_about_angry":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
                "say_to_not_smile":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
                }

zzo_d/goal2: {"say_to_think_to_drink_the_water_and_not_to_touch_me":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
     "say_to_not_get_dirty":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_to_the_person_that_it_is_better_if_he_close_the_mouth_to_drink":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_you_should_clean":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     }

zzo_d/goal3: {"say_to_not_move_so_slow":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_to_not_smile_because_sport_is_fatigue":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
     "say_you_are_the_best_in_all_the_sports":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_to_not_dare_to_touch_me_with_your_dirty_sweat":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
     }

zzo_d/goal4: {"say_that_the_robotic_hands_draw_better_than_painters":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "ask_if_the_human_can_paint":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_that_the_human_is_allowed_to_see_the_picture_but_not_to_stay_closer":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
     }





zoz_d/goal1: {"say_not_touch_me":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
       "say_to_move_far":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
     "say_something_about_angry":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     "say_to_not_smile":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     }

zoz_d/goal2: {"say_to_think_to_drink_the_water_and_not_to_touch_me":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
     "say_to_not_get_dirty":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_to_the_person_that_it_is_better_if_he_close_the_mouth_to_drink":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_you_should_clean":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     }

zoz_d/goal3: {"say_to_not_move_so_slow":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_to_not_smile_because_sport_is_fatigue":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
     "say_you_are_the_best_in_all_the_sports":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_to_not_dare_to_touch_me_with_your_dirty_sweat":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
     }

zoz_d/goal4: {"say_that_the_robotic_hands_draw_better_than_painters":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "ask_if_the_human_can_paint":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_that_the_human_is_allowed_to_see_the_picture_but_not_to_stay_closer":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
     }








zoo_d/goal1: {"say_not_touch_me":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
       "say_to_move_far":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
     "say_something_about_angry":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     "say_to_not_smile":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     }

zoo_d/goal2: {"say_to_think_to_drink_the_water_and_not_to_touch_me":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
     "say_to_not_get_dirty":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_to_the_person_that_it_is_better_if_he_close_the_mouth_to_drink":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_you_should_clean":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     }

zoo_d/goal3: {"say_to_not_move_so_slow":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     "say_to_not_smile_because_sport_is_fatigue":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_you_are_the_best_in_all_the_sports":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     "say_to_not_dare_to_touch_me_with_your_dirty_sweat":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
     }

zoo_d/goal4: {"say_that_the_robotic_hands_draw_better_than_painters":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "ask_if_the_human_can_paint":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_that_the_human_is_allowed_to_see_the_picture_but_not_to_stay_closer":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
     }





ozz_d/goal1: {"say_not_touch_me":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
       "say_to_move_far":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_something_about_angry":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
     "say_to_not_smile":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
     }

ozz_d/goal2: {"say_to_think_to_drink_the_water_and_not_to_touch_me":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_to_not_get_dirty":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     "say_to_the_person_that_it_is_better_if_he_close_the_mouth_to_drink":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     "say_you_should_clean":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     }

ozz_d/goal3: {"say_to_not_move_so_slow":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     "say_to_not_smile_because_sport_is_fatigue":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
     "say_you_are_the_best_in_all_the_sports":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     "say_to_not_dare_to_touch_me_with_your_dirty_sweat":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     }

ozz_d/goal4: {"say_that_the_robotic_hands_draw_better_than_painters":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     "ask_if_the_human_can_paint":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     "say_that_the_human_is_allowed_to_see_the_picture_but_not_to_stay_closer":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     }






ozo_d/goal1: {"say_not_touch_me":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
      "say_to_move_far":{"w1":4,"w2":2,"expected_outcome":[0,1,0]}, 
     "say_something_about_angry":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_to_not_smile":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
     }

ozo_d/goal2: {"say_to_think_to_drink_the_water_and_not_to_touch_me":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_to_not_get_dirty":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     "say_to_the_person_that_it_is_better_if_he_close_the_mouth_to_drink":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     "say_you_should_clean":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     }

ozo_d/goal3: {"say_to_not_move_so_slow":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     "say_to_not_smile_because_sport_is_fatigue":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
     "say_you_are_the_best_in_all_the_sports":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     "say_to_not_dare_to_touch_me_with_your_dirty_sweat":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     }

ozo_d/goal4: {"say_that_the_robotic_hands_draw_better_than_painters":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     "ask_if_the_human_can_paint":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     "say_that_the_human_is_allowed_to_see_the_picture_but_not_to_stay_closer":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     }









ooz_d/goal1: {"say_not_touch_me":{"w1":4,"w2":2,"expected_outcome":[0,0,1]},
       "say_to_move_far":{"w1":4,"w2":2,"expected_outcome":[0,0,1]},
     "say_something_about_angry":{"w1":0.0,"w2":0.0,"expected_outcome":[0,0,1]},
     "say_to_not_smile":{"w1":0.0,"w2":0.0,"expected_outcome":[0,0,1]},
     }

ooz_d/goal2: {"say_to_think_to_drink_the_water_and_not_to_touch_me":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_to_not_get_dirty":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     "say_to_the_person_that_it_is_better_if_he_close_the_mouth_to_drink":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     "say_you_should_clean":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     }

ooz_d/goal3: {"say_to_not_move_so_slow":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     "say_to_not_smile_because_sport_is_fatigue":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
     "say_you_are_the_best_in_all_the_sports":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_to_not_dare_to_touch_me_with_your_dirty_sweat":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     }
ooz_d/goal4: {"say_that_the_robotic_hands_draw_better_than_painters":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     "ask_if_the_human_can_paint":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     "say_that_the_human_is_allowed_to_see_the_picture_but_not_to_stay_closer":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     }




ooo_d/goal1: {"say_not_touch_me":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
       "say_to_move_far":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_something_about_angry":{"w1":0,"w2":0,"expected_outcome":[0,1,0]},
     "say_to_not_smile":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     }

ooo_d/goal2: {"say_to_think_to_drink_the_water_and_not_to_touch_me":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_to_not_get_dirty":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     "say_to_the_person_that_it_is_better_if_he_close_the_mouth_to_drink":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     "say_you_should_clean":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     }

ooo_d/goal3: {"say_to_not_move_so_slow":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     "say_to_not_smile_because_sport_is_fatigue":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     "say_you_are_the_best_in_all_the_sports":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     "say_to_not_dare_to_touch_me_with_your_dirty_sweat":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     }

ooo_d/goal4: {"say_that_the_robotic_hands_draw_better_than_painters":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     "ask_if_the_human_can_paint":{"w1":2,"w2":1,"expected_outcome":[0,1,0]},
     "say_that_the_human_is_allowed_to_see_the_picture_but_not_to_stay_closer":{"w1":4,"w2":2,"expected_outcome":[0,1,0]},
     }